### PR TITLE
Implicit Check widget color

### PR DESCRIPTION
**Pull request by wobbba**
As discussed in #615, I have implemented that the background color of the checkmark widget on implicit check (auto check) is now filled (colored background) like on manual check, but has an empty/stroked checkmark instead of a filled one. 

I have no experience with Kotlin/App-development but I tried to copy the patterns as best I could. It is a really small change and it seems to work, though I did only test it manually.

Basically, I only added the YES_AUTO to the same background color when-branch as YES_MANUAL and added an option to the RingView to make it draw the text as stroked with a stroke width dependent on the text size.

This is what it looks like now:
uhabits

### CODE REVIEW

**Comment by iSoron** [comment]
Thanks for the PR, @wobbba. I have merged it in 45a82b3c2d2f13943167ccaa84f24852aed67072.
